testing/upload_validation.ts

```diff
@@ -613,22 +613,24 @@ const scalePropertySchema = zodCompleteStrictObject<ScaleSchema & ObjectSchemaPr
   ...basePropertyValidators,
 });
 
+const optionalStringOrNumber = z.union([z.number(), z.string()]).optional();
+
 const sliderPropertySchema = zodCompleteStrictObject<SliderSchema & ObjectSchemaProperty>({
   type: zodDiscriminant(ValueType.Number),
   codaType: zodDiscriminant(ValueHintType.Slider),
-  maximum: z.number().optional(),
-  minimum: z.number().optional(),
-  step: z.number().optional(),
+  maximum: optionalStringOrNumber,
```

Comment:
do you want to validate that the only allowed string is `[Max]`?

Reply:
Oh it can be any CFL formula, the test was just an example.